README.md

```diff
@@ -451,8 +450,7 @@ Name                | Description                             | Default
 --------------------|-----------------------------------------|---------------
 [reconnect]         | Configures the exponential back off     | `{}`
 timeout             | Connect time out                        | `10000` ms
-ping                | Ping interval to test connection        | `25000` ms
-pong                | Time the server has to respond to ping  | `10000` ms
+ping                | Max time to wait for a server ping      | `45000` ms
```

Comment:
Actually this could be problematic; because these numbers are reversed, if someone set the defaults in their configuration or tweaked it a big, `timeout > ping` and thus clients will always be forcibly disconnected.

New option names + a thrown error when using the old ones would be best.